generators/mail/templates/common/manifest.xml

```diff
@@ -7,6 +7,7 @@
   <DefaultLocale>en-US</DefaultLocale>
   <DisplayName DefaultValue="<%= projectDisplayName %>" />
   <Description DefaultValue="[Outlook Add-in description]"/>
+  <SupportUrl DefaultValue="http://contoso.com/support" />
```

Comment:
No problem :) Thanks for using this tool.

I have been playing with Yo Office and found some issues with the validator itself. I am currently working on some of the changes so hopefully soon it won't complain about it. :p As of right now, the validation process is failing for me on some specific scenarios even after adding SupportUrl element to it. So at this point, I think either way could serve as a temporary workaround: a. add comment b. add SupportUrl.